docs/pyproject.toml

```diff
@@ -7,11 +7,11 @@ authors = ["ScyllaDB Documentation Contributors"]
 [tool.poetry.dependencies]
 python = "^3.7"
 pyyaml = "6.0"
-pygments = "2.2.0"
+pygments = "2.7.4"
 recommonmark = "0.7.1"
 redirects_cli ="~0.1.2"
 sphinx-scylladb-theme = "~1.4.1"
-sphinx-sitemap = "2.2.0"
+sphinx-sitemap = "2.5.0"
```

Comment:
@annastuchlik  please take a look if those versions required bumping. I took them from `pyproject.toml` from scylladb repo.

Reply:
@dgarcia360 Can you answer this question? 
The values were updated in the scylladb repo with https://github.com/scylladb/scylladb/commit/318f1f64c27e28ccbda68da60e83535c7c63f450 and https://github.com/scylladb/scylladb/commit/d0a24400238ac6e8a29e5157d7363eb2473db179, but I'm not sure it if is required in this repository.

Reply:
LGTM